operations/power_management/Shut_Down_and_Power_Off_Compute_and_User_Access_Nodes.md

```diff
@@ -12,34 +12,26 @@ System Admin Toolkit (SAT) (S-8031)* product stream documentation for instructio
 
 1. (`ncn-mw#`) List detailed information about the available boot orchestration service \(BOS\) session template names.
 
-   Identify the BOS session template names such as `cos-2.0.x`, `uan-slurm`, and choose the appropriate compute and UAN node templates for the shutdown.
+   Identify the BOS session template names such as `compute-23.7.0` and `uan-23.7.0`, and choose the appropriate compute and UAN node templates for the shutdown.
```

Comment:
I have the same comments here as I had on `operations/power_management/Power_On_and_Boot_Compute_Nodes_and_User_Access_Nodes.md`. Again, I think we should document finding the session templates in one place like `operations/power_management/Prepare_the_System_for_Power_Off.md` and reference it here.

Reply:
With this large delta to the procedures, I'm not going to include this type of change for the "finding BOS session templates".   That can be done separately in a later mod.